### ISSUE TITLE
Sync is super slow when synchronizing many small files

### ISSUE THREAD

**Issue by twatzl** [nextcloud/desktop#2125]
Hi, 

I am synchronizing about 200GB of files with a total of 70 000 files. Although they are photos they are not big (500kb to 3MB mostly). The client is now syncing for over 24 hours (it stopped in between hence not the full amount of files is displayed)

### Expected behaviour

Files should be synced in parallel and sync should be faster.

### Actual behaviour
Sync is now taking over 24 hours for just 200GB.

### Steps to reproduce
1. have many files with size between 500kb and 3MB
2. Sync
3. Wait
4. Die

### Client configuration
Client version: 
Version 2.6.2stable-Win64 (build 20191224). For more information please click here.
This release was supplied by Nextcloud GmbH
Built from Git revision 1d7455 on Dec 24 2019, 13:21:28 using Qt 5.12.5, OpenSSL 1.1.1d 10 Sep 2019

Operating system: Windows 10

![image](https://user-images.githubusercontent.com/18753370/85674812-7078d100-b6c5-11ea-8cbe-efa478545270.png)

Neither on the server nor on the client the disk usage is anywhere near maximum. Same with the network. Server load is mostly below 0.5, still free ram available on the server. Not sure about the server disk usage, but I don't think it is too high.

**Comment by er-vin** [comment]
Feels like a duplicate of #691 I'll thus close this one. Thanks for reporting though, it at least confirms it's still there and gives an extra data point.

**Comment by twatzl** [comment]
Thanks for linking the issue. 

My task manager shows that nextcloud client does not fully utilize disk or network bandwidth.
![image](https://user-images.githubusercontent.com/18753370/85676079-8e930100-b6c6-11ea-816a-83ea41603a7d.png)
![image](https://user-images.githubusercontent.com/18753370/85675945-728f5f80-b6c6-11ea-8eed-ce2cffc479e5.png)

Someone over on the other issue mentioned that Owncloud Client should be faster and indeed it is much faster and fully utilizing the available bandwidth.
![image](https://user-images.githubusercontent.com/18753370/85777686-1e19cd80-b722-11ea-9abb-d7b9aef2c054.png)
![image](https://user-images.githubusercontent.com/18753370/85777792-3b4e9c00-b722-11ea-8ab9-8d4e1275e2e1.png)

(syncing a smaller amount of files, because I did not want to resync what I already had synced. The average filesize is about the same)
![image](https://user-images.githubusercontent.com/18753370/85788478-570b6f80-b72d-11ea-9291-019e74615c53.png)
